openapi-maven-plugin/src/main/java/io/github/kbuntrock/configuration/ApiConfiguration.java

```diff
@@ -16,6 +16,9 @@ public class ApiConfiguration extends CommonApiConfiguration {
 	@Parameter
 	private String filename = "spec-open-api.yml";
 
+	@Parameter(defaultValue = "yaml")
```

Comment:
Could you put this parameter in the abstract CommonApiConfiguration? So it can be defined globally for all generated files, or override just for one file.